### PR TITLE
Started work on new template with ui-route

### PR DESCRIPTION
**Pull request by musamusa**
@tlvince, kindly review updates and merge

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <!-- Manual reference to prevent CSP error, see #110 -->
   <script src="bower_components/lodash/dist/lodash.js"></script>
   <!-- bower:js -->
-  <script src="bower_components/jquery/jquery.js"></script>
+  <script src="bower_components/jquery/dist/jquery.min.js"></script>
```
Comment (tlvince) [inline app/index.html:47]: It looks like your local jQuery is out of date. Does `bower install; npm install; grunt bower-install` revert this change?

File: app/index.html [hunk 2]
```diff
@@ -52,6 +52,7 @@
   <script src="bower_components/angular-resource/angular-resource.js"></script>
   <script src="bower_components/angular-cookies/angular-cookies.js"></script>
   <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
+  <script src="bower_components/angular-route/angular-route.min.js"></script>
```
Comment (tlvince) [inline app/index.html:55]: `angular-route` is missing from `bower.json`. You can install and save to `bower.json` in one go, e.g. with: `bower install --save angular-route`

File: app/scripts/services/inventoryfactory.js [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 angular.module('lmisChromeApp')
   .factory('inventoryFactory', function ($q, storageService, $location, $route, $rootScope) {
-
+    var common_fn = {
+        isUndefined:function (test_var){
```
Comment (tlvince) [inline app/scripts/services/inventoryfactory.js:6]: This function is unused. Nevertheless, Angular core has it built-in; [angular.isUndefined](http://docs.angularjs.org/api/ng/function/angular.isUndefined)

File: app/views/home/control-panel.html [hunk 4]
```diff
@@ -40,6 +40,7 @@
       <li><a href="">View sub-facility inventories</a></li>
       <li><a href="">View sub-facility forecasts</a></li>
       <li><a href="">View sub-facility reports</a></li>
+      <li><a ui-sref="stock_count" href="">View Daily Stock Count</a></li>
```
Comment (tlvince) [inline app/views/home/control-panel.html:43]: You don't need a empty `href` here; `ui-sref` will generate one.

File: app/scripts/services/inventoryfactory.js (hunk 3 above)
Comment (musamusa) [inline app/scripts/services/inventoryfactory.js:6]: @tlvince, I tried angular.isUndefined() and for some reason unknown to me it didn't work, so I rolled mine which worked flawlessly. I'll remove it, since I've removed the line it was used.

File: app/index.html (hunk 1 above)
Comment (musamusa) [inline app/index.html:47]: @tlvince, it did revert the change and was telling me to include the file manually.

File: app/views/home/control-panel.html (hunk 4 above)
Comment (musamusa) [inline app/views/home/control-panel.html:43]: I forgot that was there, will clean it up in my next pull request

File: app/index.html (hunk 1 above)
Comment (tlvince) [inline app/index.html:47]: @musamusa mine currently only warns me about angular-indexedDB (which I've sent a fix for https://github.com/webcss/angular-indexedDB/pull/13)
